css-positioning/relative.css

```diff
@@ -0,0 +1,39 @@
+.pokemon {
+  width: 900px;
+  margin: 0 auto;
+  text-align: center;
+}
+
+.pokemon > img {
+  width: 150px;
+}
+
+#bulbasaur {
+  position: relative;
+  top: 10px;
+  left: 10px;
+}
+
+#charmander {
+  position: relative;
+  top: 25px;
+  left: 35px;
+}
+
+#squirtle {
+  position: relative;
+  top: 50px;
+  left: 35px;
+}
+
+#pikachu {
+  position: relative;
+  top: 150px;
+  left: 35px;
+}
+
+#jigglypuff {
+  position: relative;
```

Comment:
Rather than applying `relative` individually to each rule like this, you can add it once to the `.pokemon > img` rule above and get the same effect 🤓